CRM/Member/Form/Membership.php

```diff
@@ -1950,12 +1948,7 @@ protected function legacyProcessRecurringContribution(array $params, $contactID)
     $recurParams['currency'] = $params['currency'] ?? NULL;
     $recurParams['payment_instrument_id'] = $params['payment_instrument_id'];
 
-    $recurParams['is_test'] = 0;
-    if (($form->_action & CRM_Core_Action::PREVIEW) ||
-      (isset($form->_mode) && ($form->_mode == 'test'))
```

Comment:
ensured removing $form->_action & CRM_Core_Action::PREVIEW doesn't affect the test mode processing on membership backoffice form.